aleemer/Internify
# 12B: Refactor Landing & Login Page

#### User Story
[12B Refactor Landing & Login Page](https://chaosneutral.atlassian.net/jira/software/projects/IN/boards/1/backlog?selectedIssue=IN-35)

#### Changes made
- Moved `Landing` and `Login` pages to `components` folder
- Deleted `old-components` folder
- Refactored components in `Landing` page to a simpler naming scheme
- Refactored components in `Login` page to a simpler naming scheme
- Deleted `Footer` component that resided in `molecules` folder as it already existed in `organisms`
- Refactored class names in `Footer` component to a simpler naming scheme
- Removed descendent in `Login` page to consist one less child level
    - `Login` now consists three components: `Navbar`, `LoginForm`, and `Footer` rather than `Navbar`and `LoginBody` followed by `LoginForm` and `Footer`
- Cleaned CSS for Login page
- Cleaned CSS for Landing page
- Linked the Internify logo and Sign In buttons in the Navbar to their appropriate pages

#### Does your new code introduce new warnings on dev console?
No

#### Test Steps
1. Navigate to `/` to see the `Landing` page
2. Navigate to`/login` to see the `Login` page
